### ISSUE TITLE
Can we support class weight in the CEWithChunkedOutputLoss class

### ISSUE THREAD

**Issue by ye-jin-shop** [pytorch/torchtune#1746]
I am trying to add weights to the loss function. I think it would be nice to have it in the class function? The original class CrossEntropyLoss has weight as an arg.

**Comment by joecummings** [comment]
This seems like a good idea - thanks @ye-jin-shop!

Just so I fully understand what you're trying to do, can you provide a small code example of how you would be using this weight argument?

**Comment by felipemello1** [comment]
This should be easy. We can add *args, **kwargs to the init, and something like this should work

```
class CEWithChunkedOutputLoss(torch.nn.Module):
  def __init__(self, num_output_chunks: int = 8, ignore_index: int = -100, *args, **kwargs):
          super().__init__()
          self.num_output_chunks = num_output_chunks
          self.ignore_index = ignore_index
          self.args = args
          self.kwargs = kwargs

  
      def compute_cross_entropy(
          self, logits: torch.Tensor, labels: torch.Tensor
      ) -> torch.Tensor:
          """
          Upcast logits to fp32 and compute cross entropy loss.
          """
          return F.cross_entropy(
              logits.float(), labels, ignore_index=self.ignore_index, *args, **kwargs
          )
```

i can test something like this next week, but if you want to contribute directly, i would be glad to review your PR. Let me know and i can give some pointers. Otherwise, i will post here when i get back to this

**Comment by ye-jin-shop** [comment]
Hey @felipemello1. I am having other priorities right now. I will leave this to you (not urgent, but nice to have). Thank you for your support!
